lib/banken/version.rb

```diff
@@ -1,3 +1,3 @@
 module Banken
-  VERSION = "1.0.2"
+  VERSION = '1.0.3'.freeze
```

Comment:
This PR is just cosmetic changes. so I don't think that it should update the version.